app/src/main/java/org/mozilla/fenix/library/history/HistoryController.kt

```diff
@@ -25,10 +25,14 @@ interface HistoryController {
     fun handleDeleteSome(items: Set<History>)
     fun handleRequestSync()
     fun handleEnterRecentlyClosed()
+    /**
+     * Navigates to [org.mozilla.fenix.library.syncedhistory.SyncedHistoryFragment]
+     */
+    fun handleEnterSyncedHistory()
 }
 
 @Suppress("TooManyFunctions")
-class DefaultHistoryController(
+open class DefaultHistoryController(
```

Comment:
This seems unnecessary and I don't expect anyone to override this.

Reply:
fixed